docs/developers/developer-reference/mandos-tests-reference.md

```diff
@@ -319,7 +319,7 @@ This step simulates a query to an existing smart contract. Fields:
   - `status` - indicates whether execution completed successfully or not. Status 0 means success. All errors occurring in the contract will yield status 4 (“user error”).
   - `message` (optional) - in case of error, the contract can also provide an error message. This is where this message can be checked, to make sure that the correct error occurred. It will be empty in case of success.
   - `gas` - here the consumed gas can be checked. To ignore this check, set to `“*”`
-  - `refund` - some operations, like freeing up storage actually give ERD back to the caller. To ignore this check, set to `“*”`
+  - `refund` - some operations, like freeing up storage actually gives eGLD back to the caller. To ignore this check, set to `“*”`
```

Comment:
please use `EGLD` :D

Reply:
Ah seems the spelling changed! Fixed it :)